libraries/botbuilder-dialogs-adaptive/src/input/ask.ts

```diff
@@ -49,6 +51,7 @@ export class Ask extends SendActivity implements AskConfiguration {
     public constructor(text?: string, expectedProperties?: ArrayExpression<string>) {
         super(text);
         this.expectedProperties = expectedProperties;
+        this.activity = new ActivityTemplate(text);
```

Comment:
Is `this.activity` defined on `SendActivity`? It feels like this ought to be logic that exists in the type that defines `this.activity`.

Reply:
It is, so I'll remove. [.NET does the same thing](https://github.com/microsoft/botbuilder-dotnet/blob/main/libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Input/Ask.cs#L49) and I forgot to remove the change after debugging.